### PR TITLE
Translate Chapter 3 to Spanish

### PR DESCRIPTION
**Pull request by mariagrandury**
Hey @lewtun! Here is the translation to Spanish of the remaining files of Chapter 3 (#38): 
- translate 3.3
- translate 3.3_tf
- translate 3.5
- translate 3.6
- update quiz titles for consistency across chapters
- fix some typos found in other files -> cherry-picked the commit into a separate PR https://github.com/huggingface/course/pull/511

Questions about translations:
- logits
- cased/uncased

### CODE REVIEW

**Review by lewtun — APPROVED** [review]
Thank you so much for the translation 🔥 !
